packages/tracing/src/hubextensions.ts

```diff
@@ -186,13 +186,14 @@ export function startIdleTransaction(
   transactionContext: TransactionContext,
   idleTimeout: number,
   finalTimeout: number,
+  heartbeatInterval?: number,
   onScope?: boolean,
   customSamplingContext?: CustomSamplingContext,
```

Comment:
```suggestion
  onScope?: boolean,
  customSamplingContext?: CustomSamplingContext,
  heartbeatInterval?: number,
```

We have to put this parameter at the end to not break the function call for users passing the other two optional parameters.

Reply:
Aha! You are right. I missed it, too.